project.clj

```diff
@@ -68,11 +68,12 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty9-version]
-                 [prismatic/schema "0.2.2"]
+                 [prismatic/schema "0.2.6"]
```

Comment:
no, I bumped it because I found what I thought were bugs, but left it like this because its the highest version bump we can make without breaking API and there was mention of performance improvements around the parsing of schemas related to other schemas. I'm not beholden to it, it could stay or go it doesn't matter.